Dockerfile-debian.template

```diff
@@ -1,4 +1,4 @@
-FROM buildpack-deps:jessie
+FROM buildpack-deps:name
```

Comment:
We'll need some additional steps added to the template to deal with `stretch-slim` not having gpg installed.

Why they didn't decide to have it installed in the docker image given that so many other images depend on gpg is a bit of a mystery to me.

Reply:
We could use a `buildDeps` approach and install `dirmngr ` and `gpg` before using gpg then uninstall it after with e.g. `&& apt-get purge -y --auto-remove $buildDeps`

FWIW, I think it's better to just permanently install `dirmngr` and `gpg` in the `stretch-slim` variant to make it available for down-stream usage

Reply:
YES!! Good idea!

Reply:
Might be easier to just maintain `jessie` and `stretch` templates separately since they build chain is different. When `jessie` is dropped, the template could just be deleted

Reply:
@chorrell what do you think about the common debian template?

Reply:
I think it would be simpler to have two templates rather than one.